pkg/collector/corechecks/cluster/helm/helm.go

```diff
@@ -210,10 +211,15 @@ func tagsForMetricsAndEvents(release *release, storageDriver helmStorage, includ
 	// I've found releases without a chart reference. Not sure if it's due to
 	// failed deployments, bugs in Helm, etc.
 	if release.Chart != nil && release.Chart.Metadata != nil {
+		// The helm_chart tag matches the value of the standard label helm.sh/chart
+		// https://helm.sh/docs/chart_best_practices/labels/
+		escapedVersion := strings.ReplaceAll(release.Chart.Metadata.Version, "+", "_")
```

Comment:
Should we add a test with a version that includes "+"? To avoid breaking this in the future.

Reply:
Good point 👍